webservices/rest.py

```diff
@@ -136,10 +136,10 @@ def handle_error(error):
 TRUSTED_PROXY_IPS = utils.split_env_var(env.get_credential('TRUSTED_PROXY_IPS', ''))
 # Save blocked IPs as a long string, ex. "1.1.1.1, 2.2.2.2, 3.3.3.3"
 BLOCKED_IPS = env.get_credential('BLOCKED_IPS', '')
-FEC_API_WHITELIST_IPS = env.get_credential('FEC_API_WHITELIST_IPS', False)
+FEC_API_USE_PROXY = env.get_credential('FEC_API_USE_PROXY', False)
 # Search these key_id's in the API umbrella admin interface to look up the API KEY
-DOWNLOAD_WHITELIST_API_KEY_ID = env.get_credential('DOWNLOAD_WHITELIST_API_KEY_ID')
-TRAFFIC_WHITELIST_API_KEY_IDS = env.get_credential('TRAFFIC_WHITELIST_API_KEY_IDS')
+FEC_API_DOWNLOAD_KEY_ID = env.get_credential('FEC_API_DOWNLOAD_KEY_ID')
+FEC_API_THROTTLE_BYPASS_API_KEY_IDS = env.get_credential('FEC_API_THROTTLE_BYPASS_API_KEY_IDS')
 # Settings to restrict traffic to certain API keys - only work with API umbrella
 RESTRICT_DOWNLOADS = env.get_credential('RESTRICT_DOWNLOADS', False)
```

Comment:
When do we preface with `FEC_API` and `FEC_CMS`? Update `RESTRICT_DOWNLOADS` and `RESTRICT_API_TRAFFIC` too? Is this because apps in the same space share env vars?